### ISSUE TITLE
Display current value in Skill Tree as well as current tree buff and "new" buff

### ISSUE THREAD

**Issue by Tyler799** [MechWarriorOnline/issue-tracker#109]
The Skill Tree allows you to buff certain stats. However, it doesn't show you the base stats in some cases where it very readily could. For example: AMS

I _could_ choose to take those nodes, but what is the current AMS damage? Is +1.25 significant, or is it almost nothing compared to the base value?

![currentvaluetree](https://cloud.githubusercontent.com/assets/2636178/26288384/809758f6-3e4e-11e7-8f8a-17b658a80a4e.png)

**Comment by TheFifthHorseman** [comment]
Consider the number of items affected by cooldown, heat gen, velocity etc. Showing base stats for every item is not feasible, we'd have to have an option to choose the specific weapon / piece of equipment we want to preview.

**Comment by Tyler799** [comment]
Yeah, there are some areas (like the firepower tree) where this just wouldn't work. But I think for some it might help, like AMS shown here.
